pyproject.toml

```diff
@@ -5,27 +5,24 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools_scm]
 # Including this section is comparable to supplying use_scm_version=True in setup.py.
 
-[tool.black]
+[tool.ruff]
 line-length = 88
-target-version = ['py39']
-include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs         # exclude a few common directories in the
-    | \.git          # root of the project
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-    | pb2.py
-    | \.pyi
-    | protos
-    | sdk/python/feast/embedded_go/lib
-  )/
-)
-'''
+target-version = "py39"
```

Comment:
(optional) should we target py310 in prior?

Reply:
What would be the difference? doc seems to indicate that this should be a minimum version, which is py39 now

Reply:
> (optional) should we target py310 in prior?

I can do it: I've just tried and no further changes were identified. I'm just worried about having any deprecated syntax or features that were instead allowed in py39. Is it really worth anticipating now?

Reply:
yeah it's fine, can keep now